block-util/src/archive/mod.rs

```diff
@@ -30,6 +30,8 @@ pub struct Archive {
 }
 
 impl Archive {
+    pub const MAX_MC_BLOCKS_PER_ARCHIVE: u32 = 100;
```

Comment:
We already have 

https://github.com/broxus/tycho/blob/7b6e584bfca19c7003072adb01c4361bf86e6164/core/src/storage/block/blobs/mod.rs#L70

and

https://github.com/broxus/tycho/blob/7b6e584bfca19c7003072adb01c4361bf86e6164/core/src/block_strider/provider/archive_provider.rs#L262

please use some shared const :)